<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RESPUESTA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answer_provided">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="check_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPROBAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="math_problem">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="time_left">
    <property name="geometry">
     <rect>
      <x>747</x>
      <y>60</y>
      <width>41</width>
      <height>401</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="user_feedback">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="difficulty_level">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel de Dificultad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>COMENZAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
